account_credit_control/tests/test_account_invoice.py

```diff
@@ -6,47 +6,46 @@
 from odoo import fields
 from odoo.tests.common import TransactionCase
 from odoo.exceptions import UserError
+from odoo.tests import tagged
 
 
+@tagged('post_install', '-at_install')
 class TestAccountInvoice(TransactionCase):
-    post_install = True
-    at_install = False
 
     def test_action_cancel(self):
         """
         Test the method action_cancel on invoice
         We will create an old invoice, generate a control run
         and check if I can unlink this invoice
-        :return:
         """
         journal = self.env['account.invoice']._default_journal()
 
         account_type_rec = self.env.ref('account.data_account_type_receivable')
-        account = self.env['account.account'].create({
+        account = self.env['account.account'].create([{
```

Comment:
Why adding batch create?

Reply:
because `account.account` uses the batch create

Reply:
But the use is transparent. You don't need to call it in a list AFAIK

Reply:
If you don't call it in a list, then the api of odoo will put it in a list before doing the create. It's the same.

Reply:
Yes, that's why I ask you to not add ~40 lines of diff because of this, as it adds no value.

Reply:
The lines affected because of this are fewer than 40 if I remember correctly. It's surprising me that you are trying to justify your point just by mentioning this few lines diff when this PR has a LOT of lines diff due to other reasons, the most part due to putting each attribute of a declared field in a different line.

Reply:
My justification is the same always: don't touch what is not needed to be touched unless there's a good reason for doing it, and this one is not.

Reply:
The change of putting each attribute of a declared field in a different line it's not needed, but it's done for readability reasons. This case is the same, with a list, you as a reader, understands that `account.account` model has by default the creation batch and if in the future a new tests is added, then the people adding it may have that in mind. If not, then the people will always use the create without the batch.
But for example, the `account.invoice` model doesn't have the creation batch by default. Thus, in the cases of creating an `account.invoice` in this module, you can see that they are created with a dict, without applying the batch.

Reply:
No, this change is not for readability. It's totally unneeded. There's no difference between passing a list or only one value. The implementation might change and won't affect this.  Other thing is if you create several accounts, then you can benefit from do all in batch.

Reply:
Reverted the changes as you mentioned. You should be happy I always end doing what you want xD

Reply:
Yes, very happy :smiley: . Your contributions are of course valuable, and I always appreciate a constructive dialog.

Reply:
I count 132 lines of diff in the last forced push

More than 40! :stuck_out_tongue_winking_eye:

Reply:
You are duplicating and thus counting wrong. 68 additions and 68 deletions. It means 68 lines changed!

BTW, when I said less that 40, I was referring only to the `account.account` ones.

Reply:
No, that's an important thing! In fact, you have 132 lines of diff, 66 removing a line, and 66 adding another one "slightly different", but this something that git doesn't matter. In terms of commit size, it counts double. Each change means 2 diff lines.

Reply:
good to know